pyramid-app/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/db.py

```diff
@@ -0,0 +1,40 @@
+import sqlalchemy
```

Comment:
This `db.py` template is only used when using the cookiecutter to generate a new Pyramid app with `postgres=yes`. Running `make template` in existing projects won't update `db.py`